test/FactSkill/fact-skill-tests.common.yml

```diff
@@ -19,8 +19,10 @@ configuration:
     - A phrase
   - shouldEndSession: false
 - "Get New Facts":
-  - prompt == "Here's your fact"
-  - cardTitle  =~   Space Facts # Has extra spaces - need to be handled correctly
+  - prompt == "here's your fact"
+  - transcript == "here's your fact"
+  - cardTitle  =~ mainTitle # Has extra spaces - need to be handled correctly
+  - cardTitle  =~ maintitle # Has extra spaces - need to be handled correctly
```

Comment:
duplicated line

Reply:
They are not the same - one has a capital "T".

I update the test with a better comment and also changed slightly to make it more clear what is happening.